RecoEgamma/EgammaPhotonAlgos/src/ConversionTrackFinder.cc

```diff
@@ -1,6 +1,8 @@
 //
 #include "RecoEgamma/EgammaPhotonAlgos/interface/ConversionTrackFinder.h"
 //
+#include "Geometry/CommonDetUnit/interface/GlobalTrackingGeometry.h"
```

Comment:
similar to ConversionSeedFinder.cc: is this include needed here?

Reply:
yes, I think so (in any case it will come in via the include for TrackerGeometry, so its a bit hard to tell)